installer/x86_64/install.sh

```diff
@@ -92,6 +92,39 @@ VAR_LOG_SIZE=4096
 
 [ -r platforms/$onie_platform ] && . platforms/$onie_platform
 
+# Verify image platform is inside devices list
+# Temporarily set TIMEOUT as 10 seconds
+if [ "$install_env" = "onie" ] && [ -d devices ]; then
+    if [ ! -f devices/$onie_platform ]; then
+        TIMEOUT=10
+        echo "The image you're trying to install is of a different platform as the running plaform"
+        while true; do
+            read -t $TIMEOUT -r -p "Do you still wish to install this image?[Y/n]" input
```

Comment:
>Y [](http://example.com/codeflow?start=77&length=1)

If you use uppercase `Y`, it normally means the default when user input 'Enter' directly.

Reply:
will change to [y/n] later